spring-integration/spring-boot-autoconfigure/src/test/java/ai/timefold/solver/spring/boot/autoconfigure/TimefoldSolverMultipleSolverAutoConfigurationTest.java

```diff
@@ -297,6 +298,7 @@ void solve() {
     }
 
     @Test
+    @ResourceLock("yaml")
```

Comment:
Ah, I see! I fixed several of the same issue, and I clearly missed one spot.
I fixed it in other places by merging the two test methods into one; has the same effect as your solution, but will less lines of code and arguably no worse to read. Let's apply for consistency, please.